bin/rspecq

```diff
@@ -91,6 +96,7 @@ opts[:file_split_threshold] ||= Integer(ENV["RSPECQ_FILE_SPLIT_THRESHOLD"] || 99
 opts[:report] ||= env_set?("RSPECQ_REPORT")
 opts[:report_timeout] ||= Integer(ENV["RSPECQ_REPORT_TIMEOUT"] || DEFAULT_REPORT_TIMEOUT)
 opts[:max_requeues] ||= Integer(ENV["RSPECQ_MAX_REQUEUES"] || DEFAULT_MAX_REQUEUES)
+opts[:fail_fast] ||= Integer(ENV["RSPECQ_FAIL_FAST"]) rescue nil
```

Comment:
Let's make the default `false` here, since that's the default for consistency with `Queue`.

Reply:
Sorry, I mean `0`, not `false`.

Reply:
So that would be consistent with the other options too:

```ruby
DEFAULT_FAIL_FAST = 0

# ...
opts[:fail_fast] || Integer(ENV["RSPECQ_FAIL_FAST"] || DEFAULT_FAIL_FAST)
```